### PR TITLE
added loading basic info on product page + refactoring

### PR DESCRIPTION
**Pull request by gvodianytskyi**

### CODE REVIEW

File: src/components/delivery-details/actions/delivery-details.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import config from '../../../config';
-
 import {
   DELIVERY_DETAILS_FAILED,
   DELIVERY_DETAILS_SUCCESS,
   DELIVERY_DETAILS_REQUEST,
 } from '../delivery-statuses';
 
+const deliveryDetailsUrl = 'http://private-880ca-test12906.apiary-mock.com/delivery-details/1';
```
Comment (l0lka) [inline src/components/delivery-details/actions/delivery-details.js]: I suppose the link should be like http://private-880ca-test12906.apiary-mock.com/delivery-details/ (without predefined id)

File: src/components/delivery-details/actions/delivery-details.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ export const deliveryDetailsRequest = () => ({
 
 export const fetchDeliveryDetails = () => (dispatch) => {
   dispatch(deliveryDetailsRequest());
-  return fetch(config.deliveryDetailsUrl)
+  return fetch(deliveryDetailsUrl)
```
Comment (l0lka) [inline src/components/delivery-details/actions/delivery-details.js]: Accordingly somewhere here should be passed id...?

File: src/components/delivery-details/actions/delivery-details.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ export const deliveryDetailsRequest = () => ({
...
     .then(response => response.json())
     .then(response => dispatch(deliveryDetailsSuccess(response)))
```
Comment (serhiirubets) [inline src/components/delivery-details/actions/delivery-details.js]: We have added service api for this logic, may you use it ?

File: src/components/product-page/actions/product-page.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { get } from '../../../common/services/api-service';
+
+const productUrl = 'https://private-bf0eb-test12906.apiary-mock.com/product/';
```
Comment (l0lka) [inline src/components/product-page/actions/product-page.js]: if you started to put links in config file, so may be better to move there this link as well

File: src/components/products-list/actions/products-list.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
-import ApiService from '../../../common/services/api-service';
+import { get } from '../../../common/services/api-service';
 
-export const productList = () => dispatch => ApiService.get('https://private-bf0eb-test12906.apiary-mock.com/products-list', dispatch);
+export const productList = () => dispatch => get('https://private-bf0eb-test12906.apiary-mock.com/products-list', 'PRODUCT_LIST', dispatch);
```
Comment (l0lka) [inline src/components/products-list/actions/products-list.js]: I suppose link should not be here, may be better to put it in config

**Comment by serhiirubets** [comment]
Please, resolve conflicts
src/components/product-page/product-page.js

**Comment by sergeyamator** [comment]
Please fix trevis errors

**Pull request merged**